moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -890,29 +903,28 @@ const ProjectComponentEdit = ({
                       selectedSubcomponents={selectedSubcomponents}
                       setSelectedSubcomponents={setSelectedSubcomponents}
                     />
-                    <Grid xs={12}>
+                    <Grid item xs={12}>
                       <TextField
                         className={classes.formTextField}
                         id="moped-component-description"
                         label="Description"
                         multiline
                         rows={4}
-                        defaultValue=""
                         variant="filled"
                         value={componentDescription ?? ""}
                         onChange={e => handleDescriptionKeyDown(e)}
                         fullWidth
                       />
                     </Grid>
                     {!areMinimumFeaturesSet && (
-                      <Grid xs={12}>
+                      <Grid item xs={12}>
```

Comment:
these additions fix an error in console

Reply:
There is one more error in the console 
```
index.js:1 Warning: Failed prop type: The prop `xs` of `Grid` must be used on `item`.
    in WithStyles(ForwardRef(Grid)) (at ProjectComponentEdit.js:837)
```
However when I went to fix it, the styling/spacing ended up being changed, so I left it as is.

Reply:
👍 i took a stab at this one in my branch